<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChestXrayNodules</class>
 <widget class="qMRMLWidget" name="ChestXrayNodules">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>368</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkFittedTextBrowser" name="FittedTextBrowser">
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'PMingLiU'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Confidence Score Threshold&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="qMRMLLinearTransformSlider" name="ConfidenceScoreSlider">
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="pageStep">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>0.990000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="InferencePushButton">
     <property name="text">
      <string>Inference</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="BbxNumberSpinBox"/>
   </item>
   <item>
    <widget class="QPushButton" name="ClearBbxPushButton">
     <property name="text">
      <string>Clear Bounding Box</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="HomePagePushButton">
     <property name="text">
      <string>Home Page</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkFittedTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>ctkFittedTextBrowser.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLLinearTransformSlider</class>
   <extends>qMRMLSliderWidget</extends>
   <header>qMRMLLinearTransformSlider.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSliderWidget</class>
   <extends>ctkSliderWidget</extends>
   <header>qMRMLSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
